src/plotting/plotter.py

```diff
@@ -24,28 +23,28 @@ def __init__(self, plot_config: Dict):
 
     def create_plots(self):
         for plot_name, plot_specs in self._config.items():
-            logger.debug(f"Plotting \"{plot_name}\"")
```

Comment:
If you want, you can use the single quotes to initalise the f-string, and then use the unscaped double quotes inside.
```suggestion
logger.info(f'Plotting "{plot_name}"')
```